extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -677,6 +679,23 @@ function getClusterContexts(file: string): (() => Promise<OptionsInfo>) {
 	};
 }
 
+async function throwIfNotExistsOrNotAFile(file: string, fileDescription: string = 'kube config') {
```

Comment:
`kube config` should be localized (the config part at least I would expect to be different in other languages)

Reply:
done